app/discussion/server/methods/createDiscussion.js

```diff
@@ -86,18 +94,25 @@ const create = ({ prid, pmid, t_name, reply, users, user }) => {
 	const invitedUsers = message ? [message.u.username, ...users] : users;
 
 	const type = roomTypes.getConfig(p_room.t).getDiscussionType();
+	const description = p_room.encrypted ? '' : message.msg;
+	const topic = p_room.encrypted ? '' : p_room.name;
```

Comment:
since the parent's room name is not encrypted, I guess we can always set as `topic` .. this also improves the UX..